src/ocp_postprocess/rename_utils.rs

```diff
@@ -482,19 +482,14 @@ pub(crate) fn fix_etcd_pod_yaml_ip(pod_yaml: &str, original_ip: &str, ip: &str)
 
     let ip = if ip.contains(':') { format!("[{ip}]") } else { ip.to_string() };
 
-    let patterns = [
-        (r#"value: "https://{original_ip}:2379""#, r#"value: "https://{ip}:2379""#),
-        (r#"value: "{original_ip}""#, r#"value: "{ip}""#),
-    ];
-
-    for (pattern, replacement) in patterns {
-        pod_yaml = pod_yaml
-            .replace(
-                &pattern.replace("{original_ip}", original_ip.as_str()),
-                &replacement.replace("{ip}", ip.as_str()),
-            )
-            .to_string();
-    }
+    let new_ip_pattern = r#"{ip}"#;
+    let orig_ip_pattern = r#"{original_ip}"#;
+    pod_yaml = pod_yaml
+        .replace(
+            &orig_ip_pattern.replace("{original_ip}", original_ip.as_str()),
+            &new_ip_pattern.replace("{ip}", ip.as_str()),
+        )
+        .to_string();
```

Comment:
Is there a mistake or is this equivalent?

```suggestion
    pod_yaml = pod_yaml
        .replace(
            original_ip.as_str(),
            ip.as_str(),
        )
        .to_string();
```

Reply:
You are totally right